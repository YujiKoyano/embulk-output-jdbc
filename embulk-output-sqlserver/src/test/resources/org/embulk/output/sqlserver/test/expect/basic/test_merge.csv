ITEM1:long,ITEM2:long,ITEM3:string
10,21,aa
11,10,bb
12,20,cc
10,22,dd
