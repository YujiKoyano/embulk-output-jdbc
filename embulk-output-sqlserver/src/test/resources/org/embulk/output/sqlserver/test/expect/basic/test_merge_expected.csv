10,20,A
10,21,aa
10,22,dd
11,10,bb
11,20,C
12,20,cc
